src/services/datahub-client.ts

```diff
@@ -45,12 +47,16 @@ export interface DatahubClient {
     domainName: string;
     dateFrom: Date;
   }): Promise<TrafficSourceResult>;
-  getDailyVisitsByPeriod(query: { domainName: string; dateFrom: Date }): Promise<DailyVisitsResult>;
+  getVisitsQuantitySummarizedByPeriod(query: {
+    domainName: string;
+    dateFrom: Date;
+    periodBy: filterByPeriodOptions;
+  }): Promise<DailyAndHoursVisitsResult>;
 }
 
 export type TrafficSourceResult = ResultWithoutExpectedErrors<TrafficSource[]>;
 
-export type DailyVisitsResult = ResultWithoutExpectedErrors<DailyVisits[]>;
+export type DailyAndHoursVisitsResult = ResultWithoutExpectedErrors<DailyAndHoursVisits[]>;
```

Comment:
What do you think about also apply the new name criteria for this?

Reply:
changed